apps/root/src/pages/position-detail/components/summary-container/components/swaps/components/timeline/index.tsx

```diff
@@ -604,87 +602,89 @@ const buildModifiedRateAndDurationItem = (positionState: DCAPositionModifiedActi
+  icon: <WalletMoneyIcon size={SPACING(6)} color="inherit" />,
   content: () => {
-    const { withdrawn, generatedByYield, toPrice: oldToPrice } = positionState;
+    const intl = useIntl();
+    const [showCurrentPrice, setShowCurrentPrice] = useState(true);
+    const { withdrawn: baseWithdrawn, generatedByYield, toPrice: oldToPrice } = positionState;
     const { to } = position;
-    const yieldAmount = generatedByYield?.withdrawn;
+    const toPrice = to.price;
 
-    const currentToUsd = parseUsdPrice(to, withdrawn, toPrice);
-    const toUsd = parseUsdPrice(to, withdrawn, parseNumberUsdPriceToBigInt(oldToPrice));
-    const currentToYieldUsd = parseUsdPrice(to, yieldAmount, toPrice);
-    const toYieldUsd = parseUsdPrice(to, yieldAmount, parseNumberUsdPriceToBigInt(oldToPrice));
-
-    const showPrices = !!currentToUsd && !!toUsd;
-    const [showCurrentPrice, setShouldShowCurrentPrice] = useState(true);
-    const showYieldPrices = !!currentToYieldUsd && !!toYieldUsd;
+    const yieldAmount = generatedByYield?.withdrawn;
+    const withdrawn = baseWithdrawn - (yieldAmount || 0n);
 
-    const [showCurrentYieldPrice, setShouldShowCurrentYieldPrice] = useState(true);
+    const toUsd = parseUsdPrice(to, withdrawn, parseNumberUsdPriceToBigInt(showCurrentPrice ? toPrice : oldToPrice));
+    const toYieldUsd = parseUsdPrice(
+      to,
+      yieldAmount,
+      parseNumberUsdPriceToBigInt(showCurrentPrice ? toPrice : oldToPrice)
+    );
 
     return (
       <>
-        <Grid item xs={12}>
-          <StyledTimelineWrappedContent variant="body">
-            <FormattedMessage description="positionWithdrawn" defaultMessage="Withdraw" />
-            <CustomChip
-              icon={<ComposedTokenIcon isInChip size={4.5} tokenBottom={position.to} />}
-              pointer
-              extraText={
-                showPrices && (
-                  <DarkTooltip
-                    title={
-                      showCurrentPrice
-                        ? 'Displaying current value. Click to show value on day of withdrawal'
-                        : 'Estimated value on day of withdrawal'
-                    }
+        <ContainerBox flexDirection="column">
+          <Typography
+            variant="bodySmall"
+            fontWeight={500}
+            color={({ palette: { mode } }) => colors[mode].typography.typo3}
+          >
+            <FormattedMessage description="positionWithdrawWithdrawn" defaultMessage="Withdrawn" />
+          </Typography>
+          <ContainerBox alignItems="center" gap={2}>
+            <TokenIcon token={to} size={5} />
+            <ContainerBox flexDirection="column">
+              <ContainerBox gap={1}>
+                <Typography
+                  variant="body"
+                  fontWeight={700}
+                  color={({ palette: { mode } }) => colors[mode].typography.typo2}
+                >
+                  {formatCurrencyAmount(withdrawn, to)}
```

Comment:
In the design, this columned values are closer to each other. Maybe changing the line height?
<img width="441" alt="Screenshot 2024-03-22 at 10 02 36 AM" src="https://github.com/Mean-Finance/dca-fe/assets/144699868/0ff41d44-6fbe-4302-beb0-b025ec45d05e">